client/src/components/FileExplorer.tsx

```diff
@@ -205,12 +200,12 @@ class TreeNode extends Component<TreeNodeProps, TreeNodeState> {
   render() {
     const icon = this.props.node.children.length ? (
       this.state.childrenOpen === false ? (
-        <FontAwesomeIcon className="link-primary" icon={faFolder} />
+        <FolderFill className="link-primary" />
```

Comment:
In almost all cases, bootstrap icons need to have the `"bi"` (bootstrap icon) class added so that they align with the surrounding text.

Without `bi`:
![Screenshot 2024-06-03 at 13 25 59](https://github.com/SwissDataScienceCenter/renku-ui/assets/951086/f5ff26e9-1935-4341-a600-995b80e14a06)

WIth `bi`:
![Screenshot 2024-06-03 at 13 25 54](https://github.com/SwissDataScienceCenter/renku-ui/assets/951086/60205302-f769-464a-9997-1459daf954b2)

Here and everywhere else:
```suggestion
        <FolderFill className={cx("bi", "link-primary")} />
```